Cython/Compiler/ExprNodes.py

```diff
@@ -13,8 +13,7 @@
                Builtin=object, Symtab=object, Utils=object, find_coercion_error=object,
                debug_disposal_code=object, debug_temp_alloc=object, debug_coercion=object,
                bytearray_type=object, slice_type=object, memoryview_type=object,
-               builtin_sequence_types=object, _py_int_types=object,
-               IS_PYTHON3=cython.bint)
+               builtin_sequence_types=object, _py_int_types=object)
```

Comment:
The two different int types are gone as well.
```suggestion
               builtin_sequence_types=object)
```